packages/manager/apps/hub/src/components/tile-skeleton/TileSkeleton.component.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FunctionComponent } from 'react';
+import { OsdsSkeleton } from '@ovhcloud/ods-components/react';
+
+export const TileSkeleton: FunctionComponent = () => (
```

Comment:
You are right about the Skeletons. It doesn't make sense to call it TileSkeleton since it is not wrapped in a tile. i named it that way becaus i use it inside a tile, which is why i have 4 Skeleton. i propose creating a Skeleton component with a prop for the number of Skeletons to generate, for example <Skeleton count={4} />. What do you think?

Reply:
Not sure about that (as I'm not sure I understand the usage guidelines of the skeleton) but in my opinion your usage of skeletons depends on the layout of the component where you're using skeleton and as such I'm not sure we can have a "generic" component for it